wallet/how-to/use-sdk/3rd-party-libraries/wagmi.md

```diff
@@ -0,0 +1,137 @@
+---
+description: Integrate MetaMask SDK with Wagmi in your JavaScript dapp.
+sidebar_position: 2
+sidebar_label: Wagmi
+tags:
+  - JavaScript SDK
+---
+
+# Use Wagmi with MetaMask SDK
+
+[Wagmi](https://wagmi.sh/) is a powerful and efficient React Hooks library designed to streamline
+dapp development by simplifying Ethereum interactions.
+
+You can integrate [MetaMask SDK](../../../concepts/sdk/index.md) into your dapp alongside Wagmi,
+using the MetaMask connector with Wagmi, to enable your users to seamlessly and securely connect to
+the MetaMask browser extension and MetaMask Mobile.
+
+## Prerequisites
+
+- Ensure you have a basic understanding of Ethereum smart contracts and React Hooks.
+- Set up a project with [Wagmi](https://wagmi.sh/react/getting-started).
+- Create an Infura API key and allowlist to [make read-only requests](../../make-read-only-requests.md).
+
+## Steps
+
+### 1. Configure Wagmi with the MetaMask connector
+
+Configure Wagmi to include MetaMask as a connector and specify the Ethereum chains your dapp will support.
+For example:
+
+```javascript
+import { createConfig, http } from "wagmi";
+import { mainnet, sepolia } from "wagmi/chains";
+import { metaMask } from "wagmi/connectors";
+
+export const config = createConfig({
+    chains: [mainnet, sepolia],
+    connectors: [
+        metaMask({
+            dappMetadata: {
+                name: "Example Wagmi dapp",
+            },
+        }),
+    ],
+    transports: {
+        [mainnet.id]: http(),
+        [sepolia.id]: http(),
+    },
+});
+```
+
+### 2. Configure the SDK
+
+When configuring the connector, make sure to configure the proper
+[SDK options](../../../reference/sdk-js-options.md).
+
+```javascript
+connectors: [
+    metaMask({
+        dappMetadata: {
+            name: "Example Wagmi dapp",
+        },
+        infuraAPIKey: "YOUR-API-KEY",
+        // Other options
+    }),
+],
+```
+
+#### Dapp metadata
+
+It is required to specify the [`dappMetadata`](../../../reference/sdk-js-options.md#dappmetadata)
+option when configuring the MetaMask connector with Wagmi.
+This metadata helps identify your dapp within the MetaMask ecosystem.
+
+#### Infura API key
+
+We recommend specifying the [`infuraAPIKey`](../../../reference/sdk-js-options.md#infuraapikey)
+option to [make read-only requests](../../make-read-only-requests.md) using the Infura API.
+Read more about the [benefits of using the Infura API with Wagmi](#benefits-of-using-the-infura-api-with-wagmi).
+
+#### Universal links
+
+We do not recommend enabling the [`useDeeplink`](../../../reference/sdk-js-options.md#usedeeplink) option.
```

Comment:
Should we start this section with, "We recommend using universal links instead of instead of deeplinks..."